Cargo.toml

```diff
@@ -28,7 +28,7 @@ error-chain = "0.12"
 effective-limits = "0.5"
 flate2 = "1"
 git-testament = "0.1.4"
-home = "0.5"
+home = { git = "https://github.com/rbtcollins/home"  }
```

Comment:
Could we please have a SHA on this, so that if I decide to `cargo update` I don't have to fear messing up?

Reply:
You'd know if you did, when your rustup install went away :P.